sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/healthcare/HL7v2IOWriteIT.java

```diff
@@ -46,6 +45,8 @@
   private static final String HL7V2_STORE_NAME =
       "hl7v2_store_write_it_" + System.currentTimeMillis() + "_" + (new SecureRandom().nextInt(32));
 
+  @Rule public transient TestPipeline pipeline = TestPipeline.create();
```

Comment:
I could add 
```java
pipeline.getOptions().as(DirectOptions.class).setBlockOnRun(true);
```
But I think this is the same as `pipeline.run().waitUntilFinish();` for batch pipelines which I already use.

Reply:
`pipeline.run().waitUntilFinish()` is the correct way to wait till completion.